### ISSUE TITLE
XOP candidate scanning fails on non-text nodes sometimes

### ISSUE THREAD

**Issue by bartkl** [bartkl/robotframework-zeeplibrary#10]
Given the 'right' conditions for failure, the candidate scanning (using regex) can select nodes that are non-text and this causes the base64 decode to fail.

The fix is easy:
```python
    for el in candidates:
        if el.text:  # add this
            decoded_val = base64.b64decode(el.text)
```

**Comment by bartkl** [comment]
Example:
```xml
<ns0:StoringenReply>
    <ns0:GeconstateerdeStoring>Defect</ns0:GeconstateerdeStoring>
    <ns0:Oplossing>01</ns0:Oplossing>
</ns0:StoringenReply>
```
The problem is the divisibility by 4, which makes it a candidate.

**Comment by bartkl** [comment]
This was already fixed in a later commit (than release 0.9) in which the regex was replaced by a Pythonic way altogether.
